wp-content/plugins/core/src/Object_Meta/Object_Meta_Definer.php

```diff
@@ -30,6 +33,18 @@ public function define(): array {
 					'settings_pages' => [ $container->get( Settings\General::class )->get_slug() ],
 				] );
 			},
+
+			Post_Archive_Settings::class => static function ( ContainerInterface $container ) {
+				return new Post_Archive_Settings( [
+					'settings_pages' => [ $container->get( Settings\General::class )->get_slug() ],
+				] );
+			},
+
+			Taxonomy_Archive_Settings::class => static function (): Taxonomy_Archive_Settings {
+				return new Taxonomy_Archive_Settings( [
+					'taxonomies' => [ Category::NAME ],
```

Comment:
I did have a question about this.

Ideally, this `hero_image` field should be applied to all registered taxonomies that could potentially be viewed via the `archive.php` template. Is there a good way within the definer to grab all registered taxonomies? I'd prefer to avoid updating this every time a new taxonomy is added.

Reply:
Agreed. I think for the purposes of SquareOne defaults, we can stick to the default taxonomies included in WordPress.it will be trivial for projects to add additional supported taxonomies on an per-project basis. (Since we can't know for sure if _all_ registered taxonomies should support it.)